controller/common/invoke_hook.go

```diff
@@ -131,3 +132,12 @@ func SetWebhookTimeoutFromSchemaOrDefault(schema *v1alpha1.Webhook) webhook.Invo
 		return nil
 	}
 }
+
+// SetWebhookCABundleFromSchemaOrDefault evaluates webhook timeout and sets the
+// evaluated timeout against WebhookCaller instance
+func SetWebhookCABundleFromSchemaOrDefault(schema *v1alpha1.Webhook) webhook.InvokerOption {
```

Comment:
This does not set any default. IMO we can rename the function to `SetWebhookCABundleFromSchema`